Sources/RxCBCentral/core/RxPeripheralImpl.swift

```diff
@@ -294,9 +351,22 @@ class RxPeripheralImpl: NSObject, RxPeripheral, CBPeripheralDelegate {
         didDiscoverCharacteristicsSubject.onNext(characteristicsData)
     }
     
+    public func peripheral(_ peripheral: CBPeripheral, didDiscoverDescriptorsFor characteristic: CBCharacteristic, error: Error?) {
+            guard let desc = characteristic.descriptors else { return }
+            for des in desc {
+//                des.setValue(<#T##value: Any?##Any?#>, forKey: <#T##String#>)
+                print("BEGIN:SPAKA DESCRIPTOR========>\(des)")
+//                self.peripheral.setNotifyValue(true, for: characteristic)
+            }
+    }
+    
     /// Invoked when you retrieve a characteristic’s value, or when the peripheral notifies you the value has changed
     public func peripheral(_ peripheral: CBPeripheral, didUpdateValueFor characteristic: CBCharacteristic, error: Error?) {
         let valueData = (characteristic, error)
+        if characteristic.value != nil {
+            let value = characteristic.value?.subdata(in: Range(1...2)).withUnsafeBytes {$0.load(as: UInt8.self)}
+            print("didUpdateValueFor Characteristic", characteristic, " value:", value ?? "No data 324")
+        }
```

Comment:
Please delete, since we don't want to ship the library with live print statements